python/helios/settings.py

```diff
@@ -26,12 +27,9 @@ class ParallelizationStrategy(IntEnum):
 
 
 class LogVerbosity(IntEnum):
-    SILENT = 0b000000
-    QUIET = 0b100000
-    TIME = 0b101000
-    DEFAULT = 0b111000
-    VERBOSE = 0b111100
-    VERY_VERBOSE = 0b111111
+    DEFAULT = 0
+    VERBOSE = 1
+    VERY_VERBOSE = 2
```

Comment:
There was a reason for these weird integers: The C++ side has 6 boolean switches and they translate to the names used in the CLI in the way that the binary masks specify. The bit that connects the LogVerbosity info from Python and C++ is still missing though.